pkg/controller/clusterautoscaler/clusterautoscaler.go

```diff
@@ -81,6 +82,10 @@ func AutoscalerArgs(ca *v1.ClusterAutoscaler, cfg *Config) []string {
 		args = append(args, ScaleDownArgs(s.ScaleDown)...)
 	}
 
+	if ca.Spec.BalanceSimilarNodeGroups {
+		args = append(args, BalanceSimilarNodeGroupsArgs()...)
```

Comment:
I don't think `BalanceSimilarNodeGroupsArgs` needs to be a function. Only the bits that take a full struct or sub-section of the resource as arugments are functions here. I think in this case you can just do:
```golang
args = append(args, BalanceSimilarNodeGroupsArg.String())
```

Reply:
Done.